src/functionalTest/resources/application.yaml

```
@@ -0,0 +1,31 @@
+scheduling:
+  enabled: false
+
+spring:
+  application:
+    name: Bulk Scan Orchestrator Functional Tests
+
+queue:
+  connection-string: ${QUEUE_READ_CONN_STRING}
+  write-connection-string: ${QUEUE_WRITE_CONN_STRING}
+  read-interval: ${QUEUE_READ_INTERVAL:30000} # in ms
+
+core_case_data:
+  api:
+    url: ${CORE_CASE_DATA_API_URL:http://localhost:4452}
```

Remove env url, spring will convert to that anyway
```suggestion
    url: http://localhost:4452
```